indico/modules/events/editing/client/js/components/FileTypeModal.jsx

```diff
@@ -67,9 +67,14 @@ export default function FileTypeModal({onClose, onSubmit, fileType, header}) {
               <FinalInput
                 name="filename_template"
                 label={Translate.string('Filename template')}
-                description={Translate.string(
-                  'Glob-style filename template that all files of this type have to conform to (e.g. *_paper). No dots allowed.'
-                )}
+                description={
+                  <Translate>
+                    Glob-style filename template that all files of this type have to conform to
+                    (e.g. *_paper). No dots allowed. It is possible to use{' '}
+                    <Param name="codePlaceholder" value={'{code}'} /> as a placeholder for the
```

Comment:
I'd add `wrapper={<code />}` to the Param to make it look nicer. And the `name` can be concise e.g. just `placeholder`